### PR TITLE
Update sprint interruptions rule

### PR DESCRIPTION
**Pull request by lukecookssw**

### CODE REVIEW

File: rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ type: rule
 archivedreason: 
 title: Do you have flexibility with the order you do the work (i.e. cancelling a Sprint)?
 guid: d98a0462-7682-4532-b440-5376d2d84a44
-uri: are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint
+uri: unexpected-requests
```
Comment (JackDevAU) [inline rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md:6]: Need to change the file name as well

**Review by JackDevAU — CHANGES_REQUESTED** [review]

**Review by matt-goldman — APPROVED** [review]
LGTM 🚀

File: rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md (hunk 1 above)
Comment (lukecookssw) [reply rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md]: I assume you mean the folder? I've done that - give it a check and resolve if you're happy.

**Review by JackDevAU — APPROVED** [review]

File: rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md (hunk 1 above)
Comment (JackDevAU) [reply rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md]: yep folder!

**Pull request merged**